std-lib/bool/bool-test.lurk

```diff
@@ -0,0 +1,13 @@
+!(:load "bool.lurk")
+
+;; test that (bool_neg nil) evaluates to t
+!(:assert (bool_neg nil))
```

Comment:
The lisp way is to use dashes instead of underscores.

Reply:
Ah I was wondering about this, will change thank you